services/ui-src/src/measures/2021/WCVCH/validation.ts

```diff
@@ -0,0 +1,65 @@
+import * as PMD from "./data";
+import * as DC from "dataConstants";
+import * as GV from "measures/globalValidations";
+import { FormData } from "./types";
+import { OMSData } from "measures/CommonQuestions/OptionalMeasureStrat/data";
+
+const WCVCHValidation = (data: FormData) => {
+  const ageGroups = PMD.qualifiers;
+  const whyNotReporting = data["WhyAreYouNotReporting"];
+  const OPM = data["OtherPerformanceMeasure-Rates"];
+  const performanceMeasureArray =
+    GV.getPerfMeasureRateArray(data, PMD.data) ?? [];
+  const dateRange = data["DateRange"];
+  const deviationArray = GV.getDeviationNDRArray(
+    data.DeviationOptions,
+    data.Deviations,
+    true
+  );
+  const didCalculationsDeviate = data["DidCalculationsDeviate"] === DC.YES;
+
+  let errorArray: any[] = [];
+  if (data["DidReport"] === "no") {
+    errorArray = [...GV.validateReasonForNotReporting(whyNotReporting)];
+    return errorArray;
+  }
+
+  errorArray = [
+    ...errorArray,
+    ...GV.omsValidations({
+      data,
+      qualifiers: PMD.qualifiers,
+      categories: PMD.categories,
+      locationDictionary: GV.omsLocationDictionary(
+        OMSData(true),
+        PMD.qualifiers,
+        PMD.categories
+      ),
+      validationCallbacks: [
+        GV.validateDenominatorGreaterThanNumerator,
+        GV.validateRateNotZero,
+        GV.validateOMSTotalNDR,
+      ],
```

Comment:
Do you need this one?
```suggestion
      GV.validateRateZero,
      ],
```